docker/cache/Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV GRAPHDB_ZIP="${APP_HOME}/graphdb.zip"
 ENV REPO_CONFIG="${APP_HOME}/repo-config.ttl"
 ENV GRAPHDB_SOURCE=${GRAPHDB_SOURCE:-"${APP_HOME}/cachedata"}
 # Set GraphDB Max and Min Heap size (must be less than available RAM)
-ENV GDB_HEAP_SIZE="${GDB_HEAP_SIZE:-1g}"
+ENV GDB_HEAP_SIZE="${GDB_HEAP_SIZE:-28g}"
```

Comment:
I am not sure this actually does anything, as you are only replacing the default value here. The actual value comes from the docker-compose.yaml file. Would be interested to talk about it. I had it as 1g default so it would run on test systems, but again, the real value came from the docker-compose.yaml

Reply:
Good point I missed that. It must be working with 8g which is what is in the docker-compose.yml. I've previously set these kind of defaults just in docker-compose.yml but actually reading up on it setting them in Dockerfile and image build time might be a better option.